testing/unit/test_faucetizer.py

```diff
@@ -0,0 +1,68 @@
+"""Unit tests for Faucetizer"""
+
+import os
+import unittest
+
+import forch.faucetizer
+from forch.faucetizer import Faucetizer
+
+TEST_DATA_DIR = os.getenv('TEST_DATA_DIR')
+
+TEST_METHOD_CONFIGS = {
+    'test_faucetize_simple': {
+        'forch_config_file': 'forch_dva.yaml',
```

Comment:
I think these can just be class variables on the Test Faucetizer class.  It makes it much easier to both reference (don't need to muck about with dictionary lookup), and also override in sub-classes for specific test case setup.  The model is that additional test groups can be created by creating sub-classes of the base class, so we should use OOP design to manage things as much as possible.   See the use of CONFIG in faucet: https://github.com/faucetsdn/faucet/blob/master/tests/unit/faucet/test_valve_stack.py#L43

Reply:
I see.. Created a base class and moved the testing stuff into a new test case class that inherits the base one.